systemtest/src/test/java/io/strimzi/systemtest/metrics/MetricsST.java

```diff
@@ -385,6 +397,107 @@ void testCruiseControlMetrics() {
         }
     }
 
+    /**
+     * 1. Update metrics form whatever it is to @metricsConfigJson in spec.kafka.metrics
+     * 2. Check, whether the metrics ConfigMap is changed
+     * 3. Updates ConfigMap linked as metrics on
+     * 4. Check, whether the metrics ConfigMap is changed
+     */
+    @Deprecated
+    @Test
+    void testKafkaMetricsSettingsDeprecatedMetrics() {
```

Comment:
Should the deprecated metrics tests added also for the other components? Or do you think we do not need it?